File: Sources/SimERP.Data/DBEntities/CustomerProduct.cs
using System;
using System.Collections.Generic;

namespace SimERP.Data.DBEntities
{
    public partial class CustomerProduct
    {
        public long RowId { get; set; }
        public int CustomerId { get; set; }
        public int ProductId { get; set; }
        public int? SaleId { get; set; }
        public decimal? Price { get; set; }
        public string Notes { get; set; }
        public bool? IsActive { get; set; }
    }
}

File: Sources/SimERP/ClientApp/src/app/lists/customer/model/customerproduct.ts
import { Guid } from 'guid-typescript';

export class CustomerProduct {
    RowId: number;
    CustomerId: number;
    ProductId: number;
    ProductCode: string;
    ProductName: string = "";
    SaleId: number;
    SaleName: string = "";
    Price: number;
    UnitName: string = "";
    ProductCategoryId: Guid = null;
    ProductCategoryName: string = "";
    ProductType: number;
    Notes: string;
    IsActive: boolean = true;
    IsSale:boolean = false;
}